src/modules/test/recipes/SecurityBaselineTests_EnsureSyctl.json

```diff
@@ -3,6 +3,11 @@
     "Action": "LoadModule",
     "Module": "compliance.so"
   },
+
+  // For CBL Marriner to work
```

Comment:
>Marriner [](http://example.com/codeflow?start=13&length=8)

Typo ('Mariner', not 'Marriner')

+ question: is this for all Mariner/Azure Linux versions (1, 2 and 3) or only for some specific ones?